draft-ietf-acme-acme.md

```diff
@@ -1516,7 +1523,7 @@ authorization before any virtual servers are created and only create a certifica
 a virtual server starts up.
 
 In some cases, a CA running an ACME server might have a completely external,
-non-ACME process for authorizing a client to issue for an identifier.  In these
+non-ACME process for authorizing a client to request issuance for an identifier.  In these
```

Comment:
I think this change makes things less clear. ACME doesn't authorize who can/can't _request_ issuance, just who is authorized _to_ issue. IMO the original text is closer to reality here.

Reply:
Yeah, on second look, I agree.